types/storybook__addon-actions/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for @storybook/addon-actions 3.0
+// Type definitions for @storybook/addon-actions 3.4
 // Project: https://github.com/storybooks/storybook
-// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions by: Joscha Feth <https://github.com/joscha>, June <https://github.com/jicjjang>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.6
+// TypeScript Version: 2.9
```

Comment:
do we need to bump this? The definitions below would work for older TS as well, no?

Reply:
oh, I see. I will be fix it. Didn't i think about older TS.